src/pathfora.js

```diff
@@ -694,6 +705,40 @@
     },
 
     /**
+     * @description check if a customTrigger widget
+     * is ready to be displayed, and if so display the widget
+     * @param {object} widget
+     */
+    triggerWidget: function (widget) {
+      var valid;
+
+      for (var key in widget.watchers) {
```

Comment:
It is. I think while I was making this PR I was planning to pull it out as its own function. I can't remember why I didn't... if there's a reason, or if I forgot. I can check again though.